msi-ec.c

```diff
@@ -1780,7 +1786,11 @@ static int ec_unset_bit(u8 addr, u8 bit)
 	if (result < 0)
 		return result;
 
-	unset_bit(stored, bit);
+	if (value) {
+		set_bit(stored, bit);
+	} else {
+		unset_bit(stored, bit);
+	}
```

Comment:
```suggestion
	if (value)
		set_bit(stored, bit);
	else
		unset_bit(stored, bit);
```

No braces required here